Library/Homebrew/compat/hbc/cli/update.rb

```diff
@@ -0,0 +1,17 @@
+module Hbc
```

Comment:
Until the Homebrew Cask code is structured and the compatibility layer is a lot closer to the rest of Homebrew I disagree. Compared to the other compat modules this one is included as-is regardless of a `--no-compat` or similar, isn't included into Homebrew's `compat`.